internal/server/volume/server.go

```diff
@@ -181,6 +165,10 @@ func (s *Server) ResizeVolume(context context.Context, request *internal.ResizeV
 }
 
 func (s *Server) VolumeStats(context context.Context, request *internal.VolumeStatsRequest, version apiversion.Version) (*internal.VolumeStatsResponse, error) {
+	return nil, fmt.Errorf("DismountVolume is deprecated, use UnmountVolume")
```

Comment:
The error message is incorrect here. Also, as stated above, we want to make sure a plugin invoking VolumeStats should be able to get the desired functionality and not end up with any error.

Reply:
thanks for catching this, I think I removed the `deprecated` messages and instead all the previous requests are routed to the internal volume implementation